index.ts

```diff
@@ -1,48 +1,55 @@
 import {ApolloServer, gql} from 'apollo-server';
 
-// A schema is a collection of type definitions (hence "typeDefs")
-// that together define the "shape" of queries that are executed against
-// your data.
 const typeDefs = gql`
-    # Comments in GraphQL strings (such as this one) start with the hash (#) symbol.
 
-    # This "Book" type defines the queryable fields for every book in our data source.
-    type Book {
-        title: String
-        author: String
+    type Store {
+        id: ID!
+        name: String!
+        address: String
     }
 
-    # The "Query" type is special: it lists all of the available queries that
-    # clients can execute, along with the return type for each. In this
-    # case, the "books" query returns an array of zero or more Books (defined above).
     type Query {
-        books: [Book]
+        stores: [Store]!
```

Comment:
```suggestion
        stores: [Store!]!
```

あ、storesにはnullが入らないのでこうする必要がある

Reply:
これの意味はこのqiitaの一番下のリストがわかりやすい
https://qiita.com/NagaokaKenichi/items/d341dc092012e05d6606#%E3%83%AA%E3%82%B9%E3%83%88%E3%81%AE%E3%82%A8%E3%82%AF%E3%82%B9%E3%82%AF%E3%83%A9%E3%83%A1%E3%83%BC%E3%82%B7%E3%83%A7%E3%83%B3%E3%83%9E%E3%83%BC%E3%82%AF%E3%83%9E%E3%83%BC%E3%82%AF

Reply:
あまり意図してNullの配列にすることは多くなさそうなのと、今回の場合は入らなさそうなのでそのほうがよかったですね:+1: